### PR TITLE
[tests] rollup + karma + mocha + chai for tests 

### PR DESCRIPTION
**Pull request by adimascio**

### CODE REVIEW

File: tests/unit/data-viewer.spec.ts [hunk 1]
```diff
@@ -124,10 +126,15 @@ describe('Data Viewer', () => {
 
         const firstHeaderCellWrapper = wrapper.find('.data-viewer__header-cell');
         firstHeaderCellWrapper.trigger('click');
-        expect(firstHeaderCellWrapper.classes()).toContain('data-viewer__header-cell--active');
+        await Vue.nextTick();
+        expect(firstHeaderCellWrapper.classes()).to.include('data-viewer__header-cell--active');
+
+        // Vue.nextTick().then(() => {
```
Comment (zegonz) [inline tests/unit/data-viewer.spec.ts]: On peut supprimer ce commentaire du coup ;)

File: README.md [hunk 2]
```diff
@@ -38,12 +32,7 @@ on a change:
 yarn test:unit --watchAll
 ```
 
-`yarn test:unit` is just a shortcut for `vue-cli-service test:unit` so you
-can pass command line options to `jest` directly. For instance:
-
-```bash
-yarn test:unit tests/unit/pipebuild.spec.ts --bail
-```
+`yarn test:unit` executes concurrently rollup compilatino and karma.
```
Comment (davinov) [inline README.md]:
```suggestion
`yarn test:unit` executes concurrently rollup bundling and karma.
```

File: package.json [hunk 3]
```diff
@@ -14,53 +13,61 @@
     "playground": "concurrently \"yarn:build-bundle --watch\" \"yarn:playground:build-app --watch\" \"yarn:playground:build-server --watch\" \"node playground/server.js\"",
     "storybook": "start-storybook -p 9001 -c .storybook",
     "storybook:bundle": "rollup -c .storybook/rollup.config.js",
-    "test:unit": "vue-cli-service test:unit",
-    "tdd": "vue-cli-service test:unit --watch"
+    "test:unit": "rollup -c tests/rollup.config.test.js && karma start --single-run",
```
Comment (davinov) [inline package.json]: I don't get why it's necessary to execute rollup and then karma.
Can't just karma build the test files using the rollup-preprocessor?

Comment (adimascio) [reply package.json]: This was done in order to handle multiple entry points. I've changed that by introducing a single entry point to build the test suite, as you suggested orally, thanks!

**Comment by adimascio** [comment]
There is still a heap size issue with rollup which can be circumvented by setting `` environment variable, but this is hardly more than a dirty patch. e.g. setting max heap size to 4Gb:

```bash
export NODE_OPTIONS= "--max_old_space_size=4096"
```

**Pull request merged**